.github/workflows/nightly-build.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   compile:
+    if: github.repository == 'quic-yocto/meta-qcom-hwe'
```

Comment:
Nice, we should have the same logic applied on all jobs, not only nightly.

Reply:
updated in force push